content/docs/concepts/delivery/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Declarative Multi-Stage Delivery
+description: Learn about the core use-case of declerative multi-stage delivery.
+weight: 1
+keywords: [use-cases]
+---
+
+> **Challenge:** Today, many organization build their continuous delivery workflows by hand and in an imperative way. This can lead to a lot of manual work when a scripted delivery pipeline - with slight modifications - is spread across the organization and re-used by different teams. Just imagine the difficulty to keep all variations of this pipeline up-to-date and to consider future changes in the delivery workflow of a new artifact. 
+
+Keptn follows a *declarative multi-stage delivery* approach meaning that there is no need to write *any* imperative pipeline code. Instead, a multi-stage delivery workflow is defined declaratively by defining *what* needs to be done and all the details on *how* to achieve it is left to other components. 
```

Comment:
*any* imperative pipeline code sounds great but could be misleading.
I would say "Keptn allows to declaratively define a *multi-stage delivery workflows* by defining *what* needs to be done in the `shipyard`[LINK]. 
*How* to achieve this delivery workflow is then left to other components and also here Keptn provides deployment services, which allow you to setup a multi-stage delivery workflow without a single line of pipeline code.

Reply:
Thanks for the suggestion that makes the *what* and *how* a bit clearer. I re-used it - except of the detail about the shipyard file. Since the next pargraph is all about the shipyard, I don't want to bring it in this intro sentence.